filebeat/module/mongodb/log/_meta/fields.yml

```diff
@@ -0,0 +1,27 @@
+- name: log
+  type: group
+  description: >
+      Contains fields from MongoDB logs.
+  fields:
+  - name: timestamp
+    description: >
+        Timestamp of message
+  - name: severity
+    description: >
+        Severity level of message
+    example: I
+    type: keyword
+  - name: component
+    description: >
+        Functional categorization of message
+    example: COMMAND
+    type: keyword
+  - name: context
+    description: >
+        Context of message
+    example: initandlisten
+    type: keyword
+  - name: message
```

Comment:
We should use the common `message` field here.

Reply:
I dropped `type: text`.

Reply:
I readded `text`.